src/components/EmailForm.tsx

```diff
@@ -2,31 +2,34 @@ import { GradientSpan } from 'components/Text'
 import Button from 'components/Button'
 import Email from 'icons/Email'
 import Input from 'components/Input'
-import useEmailForm from 'hooks/useEmailForm'
 
 export default function ({
   loading,
   onSubmit,
+  value,
```

Comment:
It's better to share the state from the unifying (`EmailProof`) component, because we use this state in component unifying component itself